src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    display: block;
+    margin-right: 4px;
+    background-position: center;
+  }
+
+  &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
It's still hardcoded. It should work like in the previous task. If we specify a class with modifier `stars--2` it should be only 2 stars active, but in your case, it's always 4 stars active
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/e80be24e-583d-44dc-93bc-9c8b097253f6)